pkgs/tools/system/tuptime/default.nix

```diff
@@ -20,9 +20,14 @@ stdenv.mkDerivation rec {
     mkdir -p $out/share/man/man1
     cp src/man/tuptime.1 $out/share/man/man1/
 
-    # upstream only ships this, there are more scripts there...
-    mkdir -p $out/usr/share/doc/tuptime/contrib
-    cp misc/scripts/uptimed-to-tuptime.py $out/usr/share/doc/tuptime/contrib/
+    install -Dm 0755 $src/misc/scripts/db-tuptime-migrate-4.0-to-5.0.sh $out/share/tuptime/db-tuptime-migrate-4.0-to-5.0.sh
+
+    # no _tuptime group is created by the nixos module, only change the user
+    substituteInPlace $out/share/tuptime/db-tuptime-migrate-4.0-to-5.0.sh \
+      --replace "chown \"\''${USER_DB}\":\"\''${USER_DB}\" \"\''${SOURCE_DB}\" || exit 7" \
+        "chown \"\''${USER_DB}\" \"\''${SOURCE_DB}\" || exit 7" \
+      --replace "sqlite3" "${sqlite}/bin/sqlite3"
+      # set sqlite3 (including in a comment) to avoid using a wrapper
```

Comment:
I don't understand this comment.

Reply:
the migrate script calls `sqlite3`, i prefer not using a wrapper to set `PATH` for this one line

Reply:
changed to using a wrapper